.github/workflows/cd-syftcli.yml

```diff
@@ -0,0 +1,102 @@
+name: CD - syft-cli
+
+on:
+  schedule:
+    - cron: "00 10 * * */3" # At 10:00 UTC on every three days
+
+  workflow_dispatch:
+    inputs:
+      skip_tests:
+        description: "If true, skip tests"
+        required: false
+        default: "false"
+
+jobs:
+  call-pr-tests-linting:
+    if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+    uses: OpenMined/PySyft/.github/workflows/pr-tests-linting.yml@dev
+
+  call-pr-tests-syft:
+    if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+    uses: OpenMined/PySyft/.github/workflows/pr-tests-syft.yml@dev
+
+  call-pr-tests-stack:
+    if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+    uses: OpenMined/PySyft/.github/workflows/pr-tests-stack.yml@dev
+    secrets: inherit
+
+  # call-syft-cli-tests:
+  #   if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+  #   uses: OpenMined/PySyft/.github/workflows/pr-tests-syft-cli.yml@dev
+
+  deploy-syft-cli:
+    needs: [
+        call-pr-tests-linting,
+        call-pr-tests-syft,
+        call-pr-tests-stack,
+        #  call-syft-cli-tests,
+      ]
+    if: always() && (needs.call-pr-tests-linting.result == 'success' && needs.call-pr-tests-syft.result == 'success' && needs.call-pr-tests-stack.result == 'success' || github.event.inputs.skip_tests == 'true')
+    runs-on: ubuntu-latest
+
+    outputs:
+      current_hash: ${{ steps.get_hash.outputs.current_hash }}
```

Comment:
Q: From where do we get the current hash , because I see a step named `get-hashes` below, but the above step is `get_hashes`

Reply:
I have sanitized this @rasswanth-s we really don't need to have outputs: at the job level, since we don't have other jobs that'll need to use this value. So I have sanitized this and referred to steps instead.

Reply:
Great 🙌